.github/workflows/nextjs.yml

```
@@ -80,7 +80,7 @@ jobs:
       - name: Static HTML export with Next.js
         run: ${{ steps.detect-package-manager.outputs.runner }} next export -o docs
       - name: Upload artifact
-        uses: actions/upload-pages-artifact@v1
+        uses: actions/upload-pages-artifact@v2
```

Can't we already update it to v3? https://github.com/actions/upload-pages-artifact/releases/tag/v3.0.0
Is the `upload-pages-artifact@v3` that already switched from the deprecated `actions/upload-artifact@v3` to the newer v4.


```suggestion
        uses: actions/upload-pages-artifact@v3
```

---

@cameel we will also need to update solc-js: https://github.com/ethereum/solc-js/blob/95d3a5f953d1365388c39f60ed8d5e3ae6cabcb2/.github/workflows/scorecards.yml#L49

I will open a PR there.